src/rotation.lua

```diff
@@ -483,8 +471,8 @@ function SilentRotate:applyRotationConfiguration(rotationsTables)
             group = 'BACKUP'
         end
 
-        for index, hunterName in pairs(rotationTable) do
-            local hunter = SilentRotate:getHunter(hunterName)
+        for index, GUID in pairs(rotationTable) do
```

Comment:
How come this is a GUID? The table is built from SilentRotate:getSimpleRotationTables() which inserts the hunter’s name.
Technically speaking it will yield the same results because of the new getHunter behavior, but for clarity it's better to keep the variable name as "hunterName" is this reflects the name.
But maybe you expected to modify the behavior SilentRotate:getSimpleRotationTables() in order to insert the GUID instead of the name, and you just forgot to do so.

Reply:
I'm anticipating the #16 where `SilentRotate:getSimpleRotationTables` will indeed use GUID